qiskit/algorithms/evolvers/variational/calculators/__init__.py

```diff
@@ -0,0 +1,19 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Calculators (:mod:`qiskit.algorithms.evolvers.variational.calculators`)
+=====================================================
```

Comment:
In the context of this contribution, we do not expect users to use these calculators directly. They are more just utils. @Zoufalc Do we want to expose them in the API, what do you think?

Reply:
Thanks for pointing this out to me. 
No, I also don't think we need to expose it. In fact, we may even want to avoid implementing these "calculators". Effectively, we could avoid this concept by making use of the already existing [gradient_wrapper](https://github.com/Qiskit/qiskit-terra/blob/35c0c90c2259384a3ef7334e50fed2b186019f6b/qiskit/opflow/gradients/derivative_base.py#L73) functionality.